.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
```

Comment:
We need this on the docs, but can't add it to the repo directly because it would conflict with our  tests.